base/event.go

```diff
@@ -0,0 +1,20 @@
+package base
```

Comment:
I don't think there's anything wrong with the name _base_, but have you considered alternative names?
For instance _core_?

Reply:
Perfect!